x/stakeibc/keeper/controller_balances.go

```diff
@@ -0,0 +1,63 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/Stride-Labs/stride/x/stakeibc/types"
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+)
+
+// SetControllerBalances set a specific controllerBalances in the store from its index
+func (k Keeper) SetControllerBalances(ctx sdk.Context, controllerBalances types.ControllerBalances) {
+	store :=  prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefix(types.ControllerBalancesKeyPrefix))
+	b := k.cdc.MustMarshal(&controllerBalances)
+	store.Set(types.ControllerBalancesKey(
+        controllerBalances.Index,
+    ), b)
+}
+
+// GetControllerBalances returns a controllerBalances from its index
+func (k Keeper) GetControllerBalances(
```

Comment:
nit: this might just be my editor, but the spacing/indents in this file seem a bit off